### ISSUE TITLE
Please create a tag 'cdb' for this extension

### ISSUE THREAD

**Issue by hsato42** [CartoDB/pg_schema_triggers#7]
release 0.1 is not compatible with Postgres 9.4 and it is the only released tag for this project..

**Comment by javisantana** [comment]
I don't think this extension was created to be compatible with postgres 9.4, in any case, what's the problem ?

**Comment by hsato42** [comment]
It won't compile with 9.4.4 headers. /cc
